stock_release_channel_shipment_advice/views/stock_release_channel.xml

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2023 ACSONE SA/NV
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<odoo>
+
+    <record model="ir.ui.view" id="stock_release_channel_form_view">
+        <field name="model">stock.release.channel</field>
+        <field
+            name="inherit_id"
+            ref="stock_release_channel.stock_release_channel_form_view"
+        />
+        <field name="arch" type="xml">
+            <xpath expr="//header" position="inside">
+                <button
+                    name="button_plan_shipments"
+                    type="object"
+                    string="Plan Shipment"
+                    groups="stock.group_stock_manager"
+                    attrs="{'invisible': [('can_plan_shipment', '=', False)]}"
+                />
+            </xpath>
+            <xpath expr="//field[@name='warehouse_id']" position="after">
```

Comment:
This must be in the option tab, not in the selection criteria
```suggestion
            <xpath expr="//group[@name='options']" position="inside">
```

Reply:
As commented below, as this is not the only field, maybe put that inside a new tab instead of putting it in options